src/utils/getValidUrl.ts

```diff
@@ -1,5 +1,5 @@
 export default function getValidUrl(url: string) {
-  if (url === null || !url.startsWith("https:")) {
+  if (url === null || url === undefined || !url.startsWith("https:")) {
```

Comment:
null과 undefined를 동시에 체크하려면 아래와 같이 작성할수도 있습니다~
```suggestion
  if (url == null || !url.startsWith("https:")) {
```

Reply:
그리고 지금 다시 보니까 유효한 url인지 체크하는 함수의 내부 구현이 일반적인 상황에서의 유효한 url 검사라기 보다는 특정 맥락에서만 'https:' 라는 조건으로 url을 검사하는 것 같아서 이런 경우에는 utils에 두기 애매한 것 같아요~
해당 함수를 사용하는 컴포넌트쪽에서 선언해두고 사용해도 좋을 것 같습니다~